examples/plugin/plugin/tracer.cc

```diff
@@ -51,13 +52,15 @@ class Span final : public trace::Span
 
   bool IsRecording() const noexcept override { return true; }
 
-  Tracer &tracer() const noexcept override { return *tracer_; }
+  trace::SpanContext GetContext() const noexcept override { return span_->GetContext(); }
+  //  Tracer &tracer() const noexcept override { return *tracer_; }
 
   void SetToken(nostd::unique_ptr<context::Token> &&token) noexcept override {}
 
 private:
   std::shared_ptr<Tracer> tracer_;
   std::string name_;
+  std::unique_ptr<trace::Span> span_;
```

Comment:
It seems odd for plugin Span, which isa trace::Span by inheritance, to have a trace::Span member field, and then just use it to get the SpanContext.  Can you keep the SpanContext directly, or construct one?